docs/UserGuide.md

```diff
@@ -81,6 +83,16 @@ e.g. `07:00` is a valid time input and refers to 7am, but not `7:00`
 * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
 </box>
 
+** Notes on Fields for the students **
+
+| Field       | Prefix    | Required | Caveats                                                                                                                                                                                                                                                                                                                                                                               |
+|-------------|-----------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Name**    | `-name`   | Yes      | Names should only contain alphanumeric characters and spaces, it cannot contain special character, do note that long names might cause the Id field to be blocked, in this case you are able to drag the middle portion of the UI to reveal the ID, the upper limit of this workaround would be if the name is 220 characters long. In this case do proceed to use nicknames instead. |
```

Comment:
I feel the dragging of UI part should be moved out

Reply:
Perhaps it might be more fitting for the UI part to be added as a note/tip to specific commands such as in `view` or exams in general?